roles/system_ldap/tasks/main.yml

```diff
@@ -1,97 +1,121 @@
 ---
-# tasks file for system_ldap
-#
-- name: system_ldap | Install sssd
-  ansible.builtin.apt:
-    name: "{{ item }}"
-    state: "present"
-  loop:
-    - "{{ sssd_pkg_list }}"
-
-- name: system_ldap | Remove unwanted packages
-  ansible.builtin.apt:
-    name: "{{ item }}"
-    state: "{{ sssd__unwanted_packages_state }}"
-  loop:
-    - "{{ sssd__unwanted_packages_list }}"
-
-- name: system_ldap | change hostname to match AD 
-  ansible.builtin.command: hostnamectl set-hostname {{ host_ad_name | default(omit)}}
+- name: System_ldap | change hostname to match AD
+  ansible.builtin.command: hostnamectl set-hostname {{ host_ad_name | default(omit) }}
+  changed_when: false
   when:
     - running_on_server
 
-- name: system_ldap | allow password authentication
+- name: System_ldap | allow password authentication
   ansible.builtin.lineinfile:
     path: /etc/ssh/sshd_config
     state: present
     regexp: '^PasswordAuthentication no'
     line: 'PasswordAuthentication yes'
 
-- name: system_ldap | allow users authentication
+- name: System_ldap | allow users authentication
   ansible.builtin.lineinfile:
     path: /etc/ssh/sshd_config
     state: present
     regexp: '^AllowUsers pulsys'
     line: '# AllowUsers pulsys'
   notify: restart sshd
 
-- name: system_ldap | configure pam to enable mkhomedir
-  ansible.builtin.shell: realm permit "{{ ad_user | default('almasftp@pu.win.princeton.edu') }}"
-  become: true
-  tags:
-    - add_sftp_user
-  when:
-    - running_on_server
-  ignore_errors: true
-
-- name: system_ldap | create alma directory drop
-  ansible.builtin.file:
-    path: "{{ item }}"
-    state: directory
-    recurse: true
-    owner: "{{ almasftp_user }}"
-    group: "domain users@pu.win.princeton.edu"
-    mode: 0775
+- name: System_ldap | install necessary packages
+  ansible.builtin.apt:
+    name: "{{ item }}"
+    state: present
+    update_cache: true
   loop:
-    - "/alma/bursar"
-    - "/alma/datasync_processing"
-    - "/alma/fund_adjustment"
-    - "/alma/invoice_status"
-    - "/alma/invoices"
-    - "/alma/people"
-    - "/alma/pod"
-    - "/alma/publishing"
-    - "/alma/recap"
-    - "/alma/scsb_renewals"
```

Comment:
We still need these for building the SFTP servers, right? I don't see a change here that adds these values elsewhere. How would we retrieve them?

Reply:
this will be a separate role call lib_sftp.

Reply:
yep, maybe for now just comment them out instead of deleting them, so when we start the lib_sftp work, we can find this list easily

Reply:
they are in a local branch which I will push up... If I knew how to git... ;-)

Reply:
https://github.com/pulibrary/princeton_ansible/tree/i4938_sftp_role